internal/pkg/apperrors/errors.go

```diff
@@ -30,10 +49,14 @@ func ErrKeyNotSet(key string) (err error) {
 // GetHTTPStatusCode returns status code according to customerror and default returns InternalServer error
 func GetHTTPStatusCode(err error) int {
 	switch err {
-	case InternalServer, FailedToCreateDriver, MigrationFailure:
+	case InternalServerError, JSONParsingErrorResp, InvalidIntranetData:
 		return http.StatusInternalServerError
-	case BadRequest:
+	case OrganizationNotFound, InvalidCoreValueData, InvalidParentValue, InvalidOrgId, GradeNotFound:
```

Comment:
If invalid input is given return BadRequest.

Reply:
Done